exwm.el

```diff
@@ -98,7 +98,9 @@
   :type 'hook)
 
 (defcustom exwm-blocking-subrs
-  (list #'x-file-dialog #'x-popup-dialog #'x-select-font
+  ;; `x-file-dialog' and `x-select-font' are missing on some Emacs builds, for
+  ;; example Lucid.
+  (list 'x-file-dialog 'x-popup-dialog 'x-select-font
         #'message-box #'message-or-box)
```

Comment:
@medranocalvo @Stebalien If preferred, we can also go back to a quoted list here instead of using hashed symbols.

Reply:
I'd remove the hash from all of them, for consistency.

Reply:
Done.